tests/conditional_compilation/test_config.yml

```diff
@@ -0,0 +1,15 @@
+# Using models from https://github.com/openvinotoolkit/testdata
+# $find models -wholename "*.xml"
+
+- model:
+    path: ${SHARE}/models/mobilenet_v2_1.4_224/mobilenet_v2_1.4_224_i8.xml
```

Comment:
please replace `SHARE` with `TESTDATA` to make repository name clear

Reply:
Done